neo/test/rawiotest/common_rawio_test.py

```diff
@@ -52,21 +52,22 @@ class BaseTestRawIO:
     rawioclass = None  # the IOclass to be tested
 
     entities_to_test = []  # list of files to test compliances
-    entities_to_download = []  # when files are at gin
+    entities_to_download = []  # when files a   re at gin
```

Comment:
```suggestion
    entities_to_download = []  # when files are at gin
```
haha itchy space bar finger?